docs/user-guide/cli-arguments.md

```diff
@@ -22,6 +22,7 @@ They are set in the container spec of the `ingress-nginx-controller` Deployment
 | `--disable-sync-events` | Disables the creation of 'Sync' Event resources, but still logs them |
 | `--dynamic-configuration-retries` | Number of times to retry failed dynamic configuration before failing to sync an ingress. (default 15) |
 | `--election-id`                    | Election id to use for Ingress status updates. (default "ingress-controller-leader") |
+| `--election-ttl`                  | Duration a leader election is valid before it's getting re-elected. (Default: 30s) |
```

Comment:
Or like that:

```suggestion
| `--election-ttl`                  | Duration a leader election is valid before it's getting re-elected, e.g. `15s`, `10m` or `1h`. (Default: 30s) |
```

Reply:
much better!!

Reply:
i dont like the reading for this ` Duration a leader election is valid before it's getting re-elected` but i also dont have better suggestion :( so that is ok

Reply:
We could add a reference to the more general docs about leases and leader elections: https://kubernetes.io/docs/concepts/architecture/leases/